fix(create): bad grammar in template docs (#7978)
facebook/docusaurus: examples/classic-typescript/docs/tutorial-extras/translate-your-site.md

:::caution

> In development, you can only use one locale at a time.

Reviewer: ```suggestion
In development, you can only use one locale at a same time.
```

> In development, you can only use one locale at the same time.

Reviewer 2: Thanks

Examples shouldn't be updated, they are generated from our init templates

The fix should be done here instead: packages/create-docusaurus/templates/shared/docs/tutorial-extras/translate-your-site.md

Author: Updated the `create-docusaurus` package. Should the commit changing examples be reverted?

> In development, you can only use one locale at the same time.

Reviewer: ```suggestion
In development, you can only use one locale at a time.
```
